packages/calcite-components/src/components/fab/fab.scss

```diff
@@ -1,17 +1,33 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-fab-shadow: Specifies the shadow of the component.
+ * @prop --calcite-fab-shadow-hover: Specifies the shadow of the component when in hover state.
+ * @prop --calcite-fab-shadow-active: Specifies the shadow of the component when in active state.
+ */
```

Comment:
Should we expose background-color, border-color, corner-radius, text-color here? cc @SkyeSeitz @ashetland @alisonailea

Reply:
Are we doing that for Button? Would make sense to match, if so.